__tests__/input.test.tsx

```diff
@@ -191,4 +195,20 @@ describe('Input component behaviour', () => {
         expect(globalAny.window.manywho.component.handleEvent).toHaveBeenCalled();
     });
 
+    test('on input mount when autofocusCandidate is activate', () => {
+        const componentDidMount = jest.spyOn(Input.prototype, 'componentDidMount');
+        inputWrapper = manyWhoMount(
+            'ContentString',
+            null,
+            true,
+            null,
+            false,
+            null,
+            true,
+        );
+        expect(componentDidMount).toHaveBeenCalled();
```

Comment:
Not sure if it is necessary to assert that Reacts lifecycle methods get called. As really we are then just testing an external libraries functionality.

Reply:
Yeah, you are right it doesn't bring to much value to the test, I have changed the test to verify that the focus happens after the element is mounted.